spec/spec_helper.rb

```diff
@@ -26,7 +26,23 @@
 require "active_support"
 require "active_support/core_ext"
 
+if ENV["CI"] == "true"
+  require "simplecov"
+  require "codecov"
+end
+
 RSpec.configure do |config|
+  config.before(:suite) do
+    if ENV["CI"] == "true"
+      SimpleCov.filters.clear
+      SimpleCov.start do |_file|
+        add_filter do |file|
+          !file.filename.start_with?("lib/") && !file.filename.start_with?("bin/")
```

Comment:
Add `app/` to this list.

Coming up soon, we'll need that, so start now I think.